scripts/server

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+URL=http://localhost:1313
+
+./hugo server -w -s src &
+
+sleep 2
+
+if which open 2>/dev/null 1>&2 ; then
```

Comment:
Please use `command -v` instead of `which` ([Why not use “which”? What to use then?](https://unix.stackexchange.com/q/85249/86440)).

Reply:
I'm changing it to command, we can't do "wait" in Makefile I believe.
This is also being reviewed on the ovn-website:
https://github.com/ovn-org/ovn-website/pull/4

Reply:
You can do the equivalent in a `Makefile` as follows:

```
URL := http://localhost:1313
OPEN_CMD := $(shell command -v open || command -v xdg-open || echo : 2>/dev/null)
        (sleep 2; $(OPEN_CMD) $(URL)) &
        ./hugo server -w -s src
```

Reply:
This has the advantage that it will correctly fail (from `make`’s perspective) if `hugo server` fails, which is what we really care about.